hugin/gdocs_updater.py

```diff
@@ -99,7 +100,7 @@ def reshape_run_info(self, runs, skiplist=[]):
                 p = [p]
             for project in p:
                 if len(project) == 0:
-                    project = 'Unknown, please check!'
+                    continue
```

Comment:
@senthil10 Yep. I think that is what would happen, and I guess I meant to say moved to "check status" after the due date. We would get a notification about these rare events anyway, so I don't think we need  "Unknown, please check!" projects in our list in those rare occasions. :smile: